src/adapters/rubicon.js

```diff
@@ -188,57 +194,62 @@ var RubiconAdapter = function RubiconAdapter() {
   /**
    * Define the slot using the rubicontag.defineSlot API
    * @param {Object} Bidrequest
+   * @returns {RubiconSlot} Instance of RubiconSlot
    */
   function _defineSlot(bid) {
-    _rready(function () {
-      var newSlot = window.rubicontag.defineSlot({
-        siteId: bid.params.siteId,
-        zoneId: bid.params.zoneId,
-        id: bid.placementCode,
-        sizes: bid.params.sizes
-      });
-      if (bid.params.position) {
-        newSlot.setPosition(bid.params.position);
-      }
+    var id        = bid.placementCode;
```

Comment:
if we know the type of any of these variables, it might be worth it to prepend them with the first letter of the type

i.e. nUserId, aVisitor, aKeywords, sPosition